src/ukify/ukify.py

```diff
@@ -1412,8 +1418,11 @@ def apply_config(namespace, filename=None):
         strict=False)
     # Do not make keys lowercase
     cp.optionxform = lambda option: option
-
-    cp.read(filename)
+    # If a config file is specified, try to read it after the default config file
+    if filename is not None:
+        cp.read((DEFAULT_CONFIG_FILE, filename))
```

Comment:
If a config file is given, the default config file shouldn't be used. This would just create too much confusion.

Reply:
I was under the impression that most Linux programs that load config files from multiple sources load all of them at once but in a way that allows the user defined config files to override/add on the default config.

This layout for ukify is extremely useful for me because it allows me to have a base configuration setting basic stuff like secure boot and PCR certificates, etc, then have secondary configs that set the individual kernel and initramfs for each of my layout. I currently have mainline daily drive UKI from mainline kernel and a fallback UKI based on the LTS kernel. 

I'm even considering making it so the output can be defined in the configs as well, and even allow user to set multiple config files separated by comma in the CLI.

Reply:
All systemd tools behave consistently: only one config file is read. Thus, the same should be done here for consistency.

Also note that we don't use `os.path`, but instead `pathlib.Path`. Please adjust the patch to do that too.

Reply:
Got you. I did just that and also made it so the provided filename is pathlib.Path as well. Check out #29042 since apparently, I am too stupid to do GitHub PRs.